console/console/key_generate_command.go

```diff
@@ -44,34 +43,34 @@ func (receiver *KeyGenerateCommand) Extend() command.Extend {
 // Handle Execute the console command.
 func (receiver *KeyGenerateCommand) Handle(ctx console.Context) error {
 	if receiver.config.GetString("app.env") == "production" {
-		color.Yellowln("**************************************")
-		color.Yellowln("*     Application In Production!     *")
-		color.Yellowln("**************************************")
-		color.Println(color.New(color.Green).Sprintf("Do you really wish to run this command? (yes/no) ") + "[" + color.New(color.Yellow).Sprintf("no") + "]" + ":")
+		color.Yellow().Println("**************************************")
+		color.Yellow().Println("*     Application In Production!     *")
+		color.Yellow().Printfln("**************************************")
+		color.Default().Println(color.Green().Sprintf("Do you really wish to run this command? (yes/no) ") + "[" + color.Green().Sprintf("no") + "]" + ":")
 
 		var result string
 		_, err := fmt.Scanln(&result)
```

Comment:
Can we use `ctx.Confirm` directly?

Reply:
It will be fixed in next PR. (make commands support -f flag)